minimal/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
+  "private": true,
+  "dependencies": {
```

Comment:
PR works for me on Windows 10 w/ exp cli commands `exp lan --start` followed by `exp android`

Perhaps the exp cli scripts can be added as npm scripts (for transparency to those unfamiliar with exp cli requirements)

something like:

```
  "scripts": {
    "start": "react-native-scripts start",
    "exp:start:lan": "exp start --lan",
    "exp:android": "exp android",
    "exp:ios": "exp ios",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch"
  },
```